syftlib/src/main/java/org/openmined/syft/networking/clients/SyftWebSocket.kt

```diff
@@ -24,7 +24,7 @@ class SyftWebSocket(
     keepAliveTimeout: UInt
 ) {
     @Volatile
-    var sockerStatus = AtomicBoolean(false)
+    var socketStatus = AtomicBoolean(false)
```

Comment:
I can't see `socketStatus` read anywhere else. Is it maybe a leftover?

Reply:
this can be used to decide when to create a new webSocket or as proxy to decide if socket has been initialised

Reply:
Shouldn't we add it when need it instead? It does not look as part of the API (maybe I'm wrong).